test/env.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var S = require('..');
+var $ = require('sanctuary-def');
+var type = require('sanctuary-type-identifiers');
+
+var eq = require('./internal/eq');
+
+
+var $Type = $.NullaryType(
```

Comment:
Could we name this `Type` rather than `$Type`? Regardless, let's add a type signature. :)